Mathlib/GroupTheory/MonoidLocalization.lean

```diff
@@ -61,7 +61,7 @@ this structure.
 To reason about the localization as a quotient type, use `mk_eq_monoidOf_mk'` and associated
 lemmas. These show the quotient map `mk : M → S → Localization S` equals the
 surjection `LocalizationMap.mk'` induced by the map
-`monoid_of : localization_map S (localization S)` (where `of` establishes the
+`monoid_of : localization_map S (Localization S)` (where `of` establishes the
```

Comment:
How about `localization_map` and `monoid_of`?

Reply:
@Ruben-VandeVelde , could you please have another look at this file?